spec/shortenator_spec.rb

```diff
@@ -139,6 +141,39 @@
         expect(get_bitlink_details('leafly.info/3gFjOV2')['tags']).to eq(new_tags)
       end
     end
+
+    context 'with group_guid' do
+      let(:url) { 'https://www.leafly.com/strains' }
+      let(:default_short_url) { 'https://leafly.info/2ZPPyQD' }
+      let(:default_short_text) { "text #{default_short_url}" }
+      let(:custom_group_guid) { 'Be1ojaikusR' }
+
+      context 'when not explicitly set' do
+        it 'shortens to the default group_guid' do
+          expect(subject).to eq(default_short_text)
+          expect(get_bitlink_details('leafly.info/2ZPPyQD')['references']['group']).to end_with('B01103Ajtve')
+        end
+      end
+
+      context 'when set with not the account\'s default group' do
```

Comment:
```suggestion
      context 'when provided a different group guid' do
```

The `account\'s` bothered me 😅